lib/QubitLftSyncer.class.php

```diff
@@ -92,8 +92,9 @@ private function repairEsChildrenLftValues()
         $results = QubitPdo::fetchAll($sql, $params, ['fetchMode' => PDO::FETCH_ASSOC]);
 
         $bulk = new Elastica\Bulk(QubitSearch::getInstance()->client);
-        $bulk->setIndex(QubitSearch::getInstance()->index->getName());
-        $bulk->setType('QubitInformationObject');
+        $type = 'QubitInformationObject';
+        $bulk->setIndex(QubitSearch::getInstance()->index->getIndex($type)->getName());
```

Comment:
We may need the prefix here too. Right now, it's getting the index name from ES directly but you could do something like `$bulk->setIndex(QubitSearch::getInstance()->prefix + 'QubitInformationObject');`, if you add the prefix to arElasticSearchPlugin and make it public.